src/AddTodoForm.jsx

```diff
@@ -3,15 +3,30 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
-
-
-function AddTodoForm (){
+function AddTodoForm ({ onAddTodo }){
+    const [newTodoTitle, setNewTodo] = useState('');

```

Comment:
Same here.

The title is a string, so the naming should reflect that.

For example: todoTitle and setTodoTitle, or newTodoTitle and setNewTodoTitle. However, the word “new” might not be necessary, as it’s generally implied.

Using setNewTodo can be misleading, as it suggests an object rather than a string.